<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>game_detail_deepone</class>
 <widget class="QWidget" name="game_detail_deepone">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="updateRes">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>640</y>
     <width>110</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>更新资源表</string>
   </property>
  </widget>
  <widget class="QLabel" name="resouce_version">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>670</y>
     <width>240</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>资源表版本：2.5.503</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="update_time">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>690</y>
     <width>240</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>上次更新时间：2024.10.20</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>30</y>
     <width>150</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>90</y>
     <width>150</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="review_button">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>预览</string>
   </property>
  </widget>
  <widget class="QPushButton" name="download_button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>下载</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
